packages/webcert/src/store/fmb/fmbMiddleware.test.ts

```diff
@@ -169,6 +169,18 @@ describe('Test FMB middleware', () => {
       expect(testStore.getState().ui.uiFMB.sickLeavePeriodWarning).toEqual(response.message)
     })
 
+    it('shall not fetch sick leave period warning if updated element is incomplete date range list', async () => {
+      const response = { message: 'warning message' }
+      fakeAxios.onPost('/api/fmb/validateSickLeavePeriod').reply(200, response)
+
+      testStore.dispatch(setDiagnosisListValue(getDiagnosisListValue()))
+      testStore.dispatch(updateCertificateDataElement(getIncompleteDateRangeListElement()))
```

Comment:
You can replace `getIncompleteDateRangeListElement()` with something like `fakeSickLeavePeriod({ value: list: [{ id: 'EN_FJARDEDEL', to: '2022-12-12', from: '' }] })`**